src/assets/js/data/Api.js

```diff
@@ -251,6 +251,13 @@ export const exportUsers = (eventAlias) =>
     .set('Authorization', cookies.get(CookieTypes.admin.token, {path: '/'}))
     .use(apiPrefix);
 
+
+export const generateTeams = (eventAlias) =>
+  request
+    .get('admin/generateTeams/' + eventAlias)
+    .set('Authorization', cookies.get(CookieTyeps.admin.toke, {path: '/'}))
```

Comment:
typos
```suggestion
    .set('Authorization', cookies.get(CookieTypes.admin.token, {path: '/'}))
```

Reply:
also wrap this in a `promisify(request......)` call - look at the other functions in this file